lametro/models.py

```diff
@@ -63,9 +63,21 @@ def get_queryset(self):
         when getting bill querysets. Otherwise restricted view bills
         may slip through the crevices of Councilmatic display logic.
         '''
-        filtered_qs = super().get_queryset()
+        qs = super().get_queryset()
+
+        qs = qs.exclude(
+            restrict_view=True
+        ).annotate(board_box=Case(
+            When(bill__extras__local_classification='Board Box', then=True),
```

Comment:
We used to be able to filter on `bill_type`, but that's been migrated to a [property of the Councilmatic bill object](https://github.com/datamade/django-councilmatic/blob/85e518ab6fb912a2bfa574e4e53abd82a2d1f412/councilmatic_core/models.py#L396).

You have to filter on properties in Python, which would require iterating over all bills. An operation that heavy didn't seem acceptable to me, so I wrote this annotation to approximate checking whether the property was Board Box. The weirdest thing is the check on `bill__classification`, which is an ArrayField. Docs for querying ArrayFields are here: https://docs.djangoproject.com/en/3.0/ref/contrib/postgres/fields/#contains.